bakery/env/staging.json

```diff
@@ -11,8 +11,8 @@
   "DIST_BUCKET_SAK_SECRET_NAME": "((aws-sandbox-secret-access-key))",
   "DIST_S3_BUCKET": "openstax-sandbox-web-hosting-content-gatekeeper",
   "DIST_QUEUE_STATE_S3_BUCKET": "openstax-sandbox-web-hosting-content-queue-state",
-  "DIST_ARCHIVE_QUEUE_FILENAME": "distribution-queue.json",
-  "DIST_GIT_QUEUE_FILENAME": "git-distribution-queue.json",
+  "DIST_ARCHIVE_QUEUE_FILENAME": "web-hosting-queue.json",
+  "DIST_GIT_QUEUE_FILENAME": "git-web-hosting-queue.json",
```

Comment:
Same as above. Many of these still have DIST in the name. Local env file is probably also affected.